charts/tempo-distributed/values.yaml

```diff
@@ -1303,6 +1306,12 @@ storage:
       version: null
     # -- The supported storage backends are gcs, s3 and azure, as specified in https://grafana.com/docs/tempo/latest/configuration/#storage
     backend: local
+    # -- The worker pool is used primarily when finding traces by id, but is also used by other.
+    pool:
+      # -- Total number of workers pulling jobs from the queue
+      max_workers: 400
+      # -- Length of job queue. imporatant for querier as it queues a job for every block it has to search
+      queue_depth: 20000
```

Comment:
Since these are default values, are you only wanting to expose these in the values for override?  I believe the structured config will allow the override on this already.

Reply:
yes, I only want to expose these in the values